### PR TITLE
Genesis initial balances

### PR DESCRIPTION
**Pull request by mnaamani**
Option to include initial balances for accounts at genesis from exported json file.

Corresponding export script: https://github.com/Joystream/runtime-state-export-tools/pull/1/commits/0b787aa7d6da4196351d4e3a1177a265ef0f75bb

### CODE REVIEW

**Review by bwhm — APPROVED** [review]

File: utils/chain-spec-builder/src/main.rs [hunk 1]
```diff
@@ -258,9 +287,11 @@ fn genesis_constructor(
         versioned_store_permissions_cfg,
         data_directory_config,
         content_working_group_config,
+        initial_account_balances,
     )
 }
```
Comment (shamil-gadelshin) [inline utils/chain-spec-builder/src/main.rs:292]: Please add a small comment on why we disable the lint here.

File: utils/chain-spec-builder/src/main.rs [hunk 2]
```diff
@@ -191,6 +212,7 @@ impl ChainSpecBuilder {
     }
 }
```
Comment (shamil-gadelshin) [inline utils/chain-spec-builder/src/main.rs:214]: I noticed you disabled this lint at least twice. Do you mind to convert it to module-scope lint with a comment?

File: utils/chain-spec-builder/src/main.rs [hunk 3]
```diff
@@ -241,6 +264,12 @@ fn genesis_constructor(
         )
     };
```
Comment (shamil-gadelshin) [inline utils/chain-spec-builder/src/main.rs:266]: What do you think about functional style or you prefer the imperative style more?
let initial_account_balances = initial_balances_path.map(|path| {initial_balances::from_json(path.as_path())}).unwrap_or_else(vec![]);

**Review by shamil-gadelshin — CHANGES_REQUESTED** [review]
Just a couple of minor comments.

File: utils/chain-spec-builder/src/main.rs (hunk 3 above)
Comment (mnaamani) [reply utils/chain-spec-builder/src/main.rs:266]: Applied suggestion to other code in same function as well.

File: utils/chain-spec-builder/src/main.rs (hunk 1 above)
Comment (mnaamani) [reply utils/chain-spec-builder/src/main.rs:292]: done

**Review by shamil-gadelshin — APPROVED** [review]
LGTM

**Pull request merged**